src/status_im/ui/components/chat_icon/screen.cljs

```diff
@@ -2,8 +2,8 @@
   (:require-macros [status-im.utils.views :refer [defview]])
   (:require [re-frame.core :refer [subscribe dispatch]]
             [status-im.ui.components.react :refer [view
-                                                text
-                                                image]]
+                                                   text
+                                                   image]]
             [status-im.ui.components.chat-icon.styles :as st]
```

Comment:
`:refer` -> `:as`

Reply:
no please, no, PR is huge, and there were no changes in this file, just added one function

Reply:
#3141